src/utils/chekedNewTitle.js

```diff
@@ -0,0 +1,13 @@
+const chekedNewTitle = (type, propertyNames) => {
```

Comment:
why do you call functions with past sense? checked -> check

file name shouldn't be an action, it can be, e.g. helper.js or something for specific